Documentation/templates/files/swift5.md

```diff
@@ -0,0 +1,51 @@
+## Template Information
+
+| Name      | Description       |
+| --------- | ----------------- |
+| File name | files/swift5.stencil |
```

Comment:
I'm nitpicking again, and naming is hard, but wondering if we should name that template just `swift5` like you did here, or `structured-swift5`, to mimic the naming we've been using for the `strings` templates?

Reply:
We should definitely mirror the strings templates here. We had enough trouble with the migration from `swift3` -> `flat-swift3` back in the day.

Reply:
I was following along with the convention of the other templates here. They all start like that (`fonts/swift5.stencil`, etc.)

Reply:
The doc format is ok, the question is more about how should we name the stencil template itself 🙂 and I agree with David, I think we should rename the template file `structured-swift5.stencil` (and of course update that line in the documentation accordingly then)